### ISSUE TITLE
ros service list; doesn't list most services

### ISSUE THREAD

**Issue by felixsanz** [rancher/os#1665]
```
$ sudo ros s list
disabled amazon-ecs-agent
disabled kernel-extras
disabled kernel-headers
disabled kernel-headers-system-docker
disabled open-vm-tools
disabled zfs
```

And thats all.

When doing `sudo ros s pull`, a lot of containers are pulled, for example console, ntp, docker, personal services that i've installed, etc. But `list` only show those from above. It shouldn't be consistent between commands?

**RancherOS Version: (ros os version)**

0.8.1

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**

baremetal

**Comment by ket4yii** [comment]
I have the same problem

**Comment by SvenDowideit** [comment]
yup, i agree too - and then there are some magical internal services that happen too

I started working on chancing the CLI to do things like this, but its going to be a pretty big change.

at the moment - `ros service list` only lists the external servuces`
